ouroboros-consensus/src/unstable-mock-block/Ouroboros/Consensus/Mock/Protocol/Praos.hs

```diff
@@ -1,3 +1,5 @@
+{-# OPTIONS_GHC -Wno-orphans            #-}
```

Comment:
Move this after the LANGUAGE pragmas.

Also if you could annotate in a comment why we need it?

Reply:
Will no longer need this.